kuma/dashboards/tests/test_views.py

```diff
@@ -74,23 +74,28 @@ def known_author(wiki_user):
     return wiki_user
 
 
-def test_revisions(root_doc, client):
-    """The revision dashboard works."""
+def test_revisions_not_logged_in(root_doc, client):
+    """A user who is not logged in can't see the revisions dashboard."""
     response = client.get(reverse('dashboards.revisions'))
+    assert response.status_code == 302
```

Comment:
I'd vote for adding something like:
```python
assert 'en-US/users/signin?' in response['Location']
```
after this line, just to make it clear what's going on -- like we did [here](https://github.com/mozilla/kuma/blob/9589b848ff098cce4c4827fc15668e4bae4d3c5a/kuma/wiki/tests/test_views_admin.py#L37).

Reply:
```suggestion
    assert response.status_code == 302
    assert 'en-US/users/signin?' in response['Location']
```

Reply:
Done